src/sass/mui/_select.scss

```diff
@@ -117,4 +117,16 @@ $xFormLabelLineHeight: floor($mui-label-font-size * 1.25);
       background-color: mui-color('grey', '200');
     }
   }
+
+  > .optgroup-option {
+    text-indent: 1em;
+  }
+
+  > .optgroup-label {
```

Comment:
This would be nice. Also, I think we can fix the background color of the group labels so as to provide visual aids in the form of headers?